transport/src/main/java/io/servicefabric/transport/protocol/ProtostuffMessageSerializer.java

```diff
@@ -0,0 +1,34 @@
+package io.servicefabric.transport.protocol;
+
+import static io.protostuff.ProtostuffIOUtil.writeTo;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufOutputStream;
+import io.netty.handler.codec.EncoderException;
+import io.protostuff.Schema;
+import io.protostuff.runtime.RuntimeSchema;
+import io.servicefabric.transport.utils.RecyclableLinkedBuffer;
+
+public final class ProtostuffMessageSerializer implements MessageSerializer {
+
+	private static final RecyclableLinkedBuffer recyclableLinkedBuffer = new RecyclableLinkedBuffer();
+
+	// TODO [Anton Kharenko]: Move to more appropriate place
+	static {
```

Comment:
Correct, but why to store/get MessageSchema each time  if we could know its instance upfront?